src/wp-includes/admin-bar.php

```diff
@@ -287,9 +287,8 @@ function wp_admin_bar_my_account_item( $wp_admin_bar ) {
 			'title'  => $howdy . $avatar,
 			'href'   => $profile_url,
 			'meta'   => array(
-				'class'      => $class,
-				/* translators: %s: Current user's display name. */
-				'menu_title' => sprintf( __( 'Howdy, %s' ), $current_user->display_name ),
+				'class'      => ( false === $avatar ) ? '' : 'with-avatar',
+				'menu_title' => wp_strip_all_tags( $howdy ),
```

Comment:
Why `wp_strip_all_tags()`?

Reply:
`menu_title` have `sprintf( __( 'Howdy, %s' ), $current_user->display_name )` string without `span`

Reply:
I see. You're reusing the `$howdy` from above.

Reply:
Yes